src/App.js

```diff
@@ -34,28 +40,38 @@ export default class App extends React.Component{
   }
 
   render () {
-    return(
-      <Router>
-        <Navbar />
-        <Switch>
-          <Route path='/login'>
-            <LoginForm onSubmit={this.login}/>
-          </Route>
-          <PrivateRoute path='/Rooms' isLoggedIn={this.state.loggedIn}>
-            <Rooms />
-          </PrivateRoute>
-          <PrivateRoute path='/Speakers' isLoggedIn={this.state.loggedIn}>
-            <Speakers />
-          </PrivateRoute>
-          <PrivateRoute path='/Times' isLoggedIn={this.state.loggedIn}>
-             <Times />
-          </PrivateRoute>
-          <PrivateRoute path='/Presentations' isLoggedIn={this.state.loggedIn}>
-            <Presentations />
-          </PrivateRoute>
-        </Switch>
-      </Router>
-    )
+  
+
+    if(this.state['loggedIn'] === false) {
+
+      return (
+        <LoginForm onSubmit={ this.login } />
+      );
+    }
```

Comment:
I'm confused as to why this if statement is here the <PrivateRoute> function handles this and will reroute to LoginForm if loggedIn is false

Reply:
For some reason when you login it doesn't reroute if this if statement is missing so for the demo I'm leaving it in but it will probably be removed once I figure out why it doesn't reroute